Automattic/wp-calypso
# Fix the padding for the first tag in the Reader

Related to #

## Proposed Changes

#91990 introduced a padding change for the first tag that looks inconsistent with the other tags as one of many styling changes 

## Why are these changes being made?

All tags look like buttons except the first one, this doesn't look good.

## Testing Instructions

1. On Desktop, go to the reader and select a tag or visit /tag/blog (or any other tag)
2. Check the right sidebar

Before the change:
<img width="1066" alt="Screenshot 2024-07-04 at 14 15 06" src="https://github.com/Automattic/wp-calypso/assets/82778/cc6206f6-0673-436f-abbb-dafb522aa679">

After the change:
<img width="1066" alt="Screenshot 2024-07-04 at 14 14 59" src="https://github.com/Automattic/wp-calypso/assets/82778/2033c979-e6d7-4503-b1f8-d2f481a279db">


## Pre-merge Checklist
